packaging/linux/run_keybase

```diff
@@ -140,8 +140,8 @@ show_cryptosquirrel() {
   [ "${KEYBASE_NO_SQUIRREL:-}" != "1" ] && cat /opt/keybase/crypto_squirrel.txt
 }
 
-warn_if_unwritable() {
-  if [ -w "$1" ]; then
+warn_if_exists_and_unwritable() {
+  if ! [ -a "$1" ] || [ -w "$1" ]; then
```

Comment:
I'm probably just dumb, but I can't find `-a` in the `man test` page, except for as the and operator.  Does it mean something else too?  From the name of the function, I would have expected `-e`.

Reply:
oh, apparently it's deprecated now (https://www.tldp.org/LDP/abs/html/fto.html, though the reference they cite is not very helpful haha). I guess because it's confusing in addition to the infix -a. Switched to -e